framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/SuiteCommand.cs

```diff
@@ -42,17 +44,28 @@ public class SuiteCommand : IConsoleCommand, ITransientDependency
         AbpNuGetIndexUrlService nuGetIndexUrlService,
         NuGetService nuGetService,
         ICmdHelper cmdHelper,
+        AuthService authService,
         CliHttpClientFactory cliHttpClientFactory)
     {
         CmdHelper = cmdHelper;
         _nuGetIndexUrlService = nuGetIndexUrlService;
         _nuGetService = nuGetService;
+        _authService = authService;
         _cliHttpClientFactory = cliHttpClientFactory;
         Logger = NullLogger<SuiteCommand>.Instance;
     }
 
     public async Task ExecuteAsync(CommandLineArgs commandLineArgs)
     {
+#if !DEBUG    
+        var loginInfo = await _authService.GetLoginInfoAsync();
```

Comment:
check the `null` case
![image](https://user-images.githubusercontent.com/9526587/193997033-dc76feea-4825-42c9-b8f0-8aebb311dcc7.png)